requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
```

Comment:
This file is a duplicates some content of the toml file. It would be easier to maintain only a single source of dependencies.  Would it be possible to use only one of the two places to define dependencies?